Source/Fuse.Triggers/Actions/Visibility.uno

```diff
@@ -31,7 +31,11 @@ namespace Fuse.Triggers.Actions
 	{
 		protected override void Perform(Node target)
 		{
-			if (target is IShow) ((IShow)target).Show();
+			var t = target.FindByType<IShow>();
```

Comment:
I really wish TriggerActions would be nodes so these could be found while rooting... scanning along the tree could be a tad expensive :/